apps/system/lockscreen/js/lockscreen_media_playback.js

```diff
@@ -94,17 +94,33 @@ LockScreenMediaPlaybackWidget.prototype = {
     }
 
     var title = metadata.title.trim();
-    var artist = metadata.artist.trim();
-    var track = [];
+    var titleNode = document.createTextNode(title || '');
+    var titleBdiNode = document.createElement('bdi');
+    titleBdiNode.appendChild(titleNode);
 
-    if (title) {
-      track.push(title);
-    }
-    if (artist) {
-      track.push(artist);
+    var artist = metadata.artist.trim();
+    var artistNode = document.createTextNode(artist || '');
+    var artistBdiNode = document.createElement('bdi');
+    artistBdiNode.appendChild(artistNode);
+
+    var emDashNode = document.createTextNode(' — '); // Using a &mdash; here.
```

Comment:
Same comments in this file. Please don't create the text nodes and bdi elements until you're in the if blocks below and are sure you'll actually need them.